requirements/README.md

```diff
@@ -0,0 +1,93 @@
+# Requirements
+During its installation, DeepChem requires various prerequisite libraries to be installed. The various `.yml` files in the `requirements` folder contain the different libraries that are the prerequisites based on the installer, operating system, etc. The different folders are used to show the dependencies of the DeepChem library on the respective deep-learning backends.
```

Comment:
How about something like this - DeepChem depends on various scientific computational libraries which are specified in the `.yml` files. The different sub-folder contains the deep-learning backend requirements of DeepChem.